### ISSUE TITLE
Unify value from noUiSlider

### ISSUE THREAD

**Issue by lionel-bijaoui** [icebob/vue-form-generator#71]
There is a few problem with noUiSlider:
- [x] It can't dynamically add or remove an handle. It can't wait for the value from the model to know if this is a double handle or not. So we need an option to explicitly tell if this is double or not (like with rangeSlider). EDIT: done in #72
- [x] When using a single handle, noUiSlider format the data like this `[Number, null]`. It expect us to use `slider.noUiSlider.get()`. So, much like with date input, we need a `formatValueToField` and `formatValueToModel` to properly format data back and forth.

I'm on both issue and I will do a PR in the next days.

**Comment by icebob** [comment]
Nice observation. Thank you

**Comment by lionel-bijaoui** [comment]
Progress: I'm done with noUiSlider, I added some dynamic CSS class and CSS fix with scale.
There is something I can't do though: if you modify the same model with noUiSlider and rangeSlider, modification of rangeSlider make noUiSlider change too, but nit the contrary.
I tried different solutions ( almost lost my mind) and was not able to avoid infinite loops. I don't know why, and I give up, since there is no real reason to use both sliders.
